Mage/Task/BuiltIn/Deployment/Strategy/GitRebaseTask.php

```diff
@@ -53,8 +53,10 @@ public function run()
         $result = $this->runCommandRemote($command);
 
         if ($result === false) {
-            $repository = $this->getConfig()->deployment('repository');
-            if ($repository) {
+            $scmConfig = $this->getConfig()->deployment('scm');
```

Comment:
Ok, so I gonna fix my typo in my PR and close yours ok? But I keep in mind your idea to make configs more clear :)